api-samples/tabCapture/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Tab Capture Example",
+  "description": "Capture a tab and play in a <video> element in a separate tab.",
+  "version": "1",
+  "manifest_version": 3,
+  "action": {
+    "default_icon": "icon.png"
+  },
+  "background": {
+    "service_worker": "background.js"
+  },
+  "permissions": ["tabs", "tabCapture"]
```

Comment:
@oliverdunk I just wanted to confirm if we require the tabs permission instead of the activeTab permission.

Reply:
Yeah, we do unfortunately, since we're calling chrome.tabs.query and expecting to get the URL of all of the tabs back.

Reply:
Gotcha! Thanks for clarifying 😄